CeeFM/Decideify
# Logged In User's Saved Suggestions now visible on all relevant content hub pages

**Description**
If the logged in user has saved suggestions, they will render in the browser now just below the suggestion request / genre submission form on every page. There are likely errors when the user doesn't have any saved suggestions, which I will handle in a near-future request.

**Type of Change**


**Testing Instructions**

- Clone this project’s repository to your machine
  -If you’ve already done this and are currently working on a different branch of the project, be sure to `git add` and `git commit` whatever you’re working on on your branch, before you attempt to switch branches
  -Then `git checkout main` to switch to the project’s main branch)
- Navigate to the root directory for this repo (yourParentFolderName/Decideify)
- `git fetch -a` to retrieve the current remote-tracked branches for the project, you should see a branch named `cm-suggestions` as one of the available branches
- `git checkout cm-suggestions` to switch to my branch
- If you don't see the most recent version of my files, `git pull origin cm-suggestions`
- `Start Decideify.sln` to open the project in Visual Studio
- If you haven’t created the Decideify database, please do so in Visual Studio and use the following SQL scripts to populate it accordingly:
  - Start a new query in your database and copy in this SQL script before running to fill out tables and columns: [First SQL Script for Creating DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Create_DB.sql)
  - ctrl + a and delete what is in that query and replace it with this SQL script before running to seed the database: [Second SQL Script for Seeding DB](https://github.com/CeeFM/Decideify/blob/main/SQL/Decideify_Seed_Users_DB.sql)

- Click the execute with debugger solid green button to launch the app
- Log in as coolin@gmail.com with password thistbh
- Go to any content hub page
- You should see 4-5 rows of saved suggestions under my account.
- Oh wait no you shouldn't because nobody has the same database as me until I update the queries to seed the database to match all of the manual testing I've been doing.
- Oh brother this fell apart fast.
- Well the first part of this would be true if you had my same database!! Come over to my house and I'll show you!

Checklist
